### DISCUSSION TITLE
8월 4일 온보딩

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/prolog#208] [Notice]
# 공통

## 프로젝트 진행 방법 소개
- 스프린트 일정
- 스프린트 계획 (https://github.com/woowacourse/prolog/wiki/%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-%EA%B3%84%ED%9A%8D)

![image](https://user-images.githubusercontent.com/46308949/127850062-545627db-0107-483a-a036-08c9d1fa73b7.png)

## 일정 소개
- 매주 월요일 5시 주간회의

## 시즌3 핵심 이슈
- 모의 면접 대비 모아보기(ex. 리포트)
- 학습로그 오버뷰 (타임라인, 태그별 모아보기)
- 이력서의 보유기술을 대신할 수 있는 페이지
- 메인 페이지 개선하기(단순 목록 대신 추가적인 정보 제공)

## 공통 인프라 소개
- 배포
- 위키

# 파트별

## 프론트엔드
- 개발 환경 및 코드컨벤션
- 현재까지의 진행상황

## 백엔드
- 도메인 소개
- cucumber
- documentation

**Comment by sunhpark42** [comment]
[프론트엔드온보딩](https://github.com/woowacourse/prolog/wiki/%ED%94%84%EB%A1%A0%ED%8A%B8%EC%97%94%EB%93%9C-%7C-%EC%8B%9C%EC%A6%8C3-%EC%98%A8%EB%B3%B4%EB%94%A9)
